joint_teapot/utils/joj3.py

```diff
@@ -82,9 +82,9 @@ def generate_scoreboard(
     submitter_row[columns.index("total")] = str(total)
 
     now = datetime.now().strftime("%Y-%m-%d %H:%M:%S")
-    submitter_row[
-        columns.index("last_edit")
-    ] = now  # FIXME: Delete this in formal version
+    submitter_row[columns.index("last_edit")] = (
+        now  # FIXME: Delete this in formal version
```

Comment:
Have you installed `pre-commit` and run it? I think the original indent is correct.

Reply:
It seems cz in my lazygit will skip pre-commit.. sorry

Reply:
fixed